state/state.go

```diff
@@ -219,8 +232,10 @@ func (s *State) Bytes() []byte {
 	return wire.BinaryBytes(s)
 }
 
-// SetBlockAndValidators mutates State variables to update block and validators after running EndBlock.
-func (s *State) SetBlockAndValidators(header *types.Header, blockPartsHeader types.PartSetHeader, abciResponses *ABCIResponses) {
+// SetBlockAndValidators mutates State variables to update block and validators after running
```

Comment:
Perhaps we could just break this into 2 almost evenly spread comments, to look even nicer?
```go
// SetBlockAndValidators mutates State variables
// to update block and validators after running EndBlock.
```